### PR TITLE
Add Node Types to deploy Braket Applications

### PR DESCRIPTION
**Pull request by FelixTruger**
This adds new Node Types, Implementations, and Artifacts needed to deploy a Braket Application.

Adds Node Types: AWS, Braket-SDK, Boto3, Braket-App
Adds corresponding Node Type Implementations for: Braket-SDK, Boto3, Braket-App
Adds Artifact Type: BraketApp
Adds Implementation/Deployment Artifacts: Braket-SDK Install, Boto3 Install, Braket-App Install, Braket-App Start, Braket-App-Adapter

### CODE REVIEW

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/AWS_w1/NodeType.tosca [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypes" id="nodeTypesQuantil-AWS_w1">
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/AWS_w1/NodeType.tosca]: This should be part of https://github.com/OpenTOSCA/tosca-definitions-common

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Boto3_latest-w1/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Boto3 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This node type installs the AWS SDK for Python (Boto3).
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Boto3_latest-w1/README.md]: Then it should also be part of the OpenTOSCA commons

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Braket-SDK [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This node type installs Braket SDK.
+
+> **⚠ Executions may not be reproducible!** This Node Type installs Braket SDK in its latest version.
+> Thus, applications that depend on this Node Type may not be compatible with the installed version.
```
Comment (lharzenetter) [inline nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md]: Maybe it's better then to install a specific version?

Did you add an example app that demonstrates it's usage to the example apps repo?

**Review by lharzenetter — CHANGES_REQUESTED** [review]

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md (hunk 3 above)
Comment (FelixTruger) [reply nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md]: Installing a specific version could also cause troubles with reproducibility, as it's likely outdated when the next guy wants to use this node type. Is it possible to draw the version from a property and install latest just in the case that no specific version has been set via that property?

I guess the example app should be added to [tosca-definitions-qc-applications](https://github.com/UST-QuAntiL/tosca-definitions-qc-applications)?

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/AWS_w1/NodeType.tosca (hunk 1 above)
Comment (FelixTruger) [reply nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/AWS_w1/NodeType.tosca]: Moved the AWS and Boto3 Node Types/Implementation [here](https://github.com/OpenTOSCA/tosca-definitions-common/pull/39).

File: nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md (hunk 3 above)
Comment (lharzenetter) [reply nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/Braket-SDK_latest-w1/README.md]: Okay, then just add this explanation to the readme.

Yes, example apps should be in the qc-apps repo.

File: artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/Braket-App-Adapter_DA-w1/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Braket App Adapter DA [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
```
Comment (lharzenetter) [inline artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/Braket-App-Adapter_DA-w1/README.md:2]: Here a short description should be added about what the adapter is capable of

File: artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/Braket-App_Start-w1/files/braket-app_start.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo 'Port:'$Port
+export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+export AWS_REGION=$AWS_REGION
+
+nohup python3 braket_app/adapter.py $Port > braket_app/output.log &
```
Comment (lharzenetter) [inline artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/Braket-App_Start-w1/files/braket-app_start.sh]: If you add something like [here](https://github.com/OpenTOSCA/tosca-definitions-common/blob/main/artifacttemplates/http%253A%252F%252Fopentosca.org%252Fartifacttemplates/SpringWebApp_Start-w1/files/start.sh#L4) debugging becomes easier as 
1. the error output is also redirected to the log by adding `2>&1`
2. the process id of the started application is logged by performing `echo $! > braket-app.pid`

File: artifacttypes/https%3A%2F%2Fust-quantil.github.io%2Fartifacttypes/BraketApp/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# BraketApp Artifact Type [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This artifact type is used to identify Qiskit applications.
```
Comment (lharzenetter) [inline artifacttypes/https%3A%2F%2Fust-quantil.github.io%2Fartifacttypes/BraketApp/README.md]: Qiksit? I guess BraketApps?

**Review by lharzenetter — CHANGES_REQUESTED** [review]
some minor improvements

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**